### PR TITLE
Milestone: Mobile Menu 

### PR DESCRIPTION
**Pull request by Roseokpe**
In this milestone, I created mobile menu with html, Css and Js.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
+const hamburger = document.querySelector('.hamburger');
+ const menuitems = document.querySelector('.menu-list');
+// const aboutContent = document.querySelector('');
+// const modal = document.querySelector('');
+  hamburger.addEventListener('click', () =>{
+  //hamburger.classList.toggle('active');
+  document.querySelector('.mobile-menu').style.display = 'block';
```
Comment (sja-thedude) [inline index.js]: - [ ] Kindly remove the commented code for clean code purposes.

File: style.css [hunk 2]
```diff
@@ -329,6 +329,75 @@ hr {
...
+}
+
+.active .bar-1 {
+  transform: rotate(-45deg) translate(-7px, 5px);
+}
+
+.active .bar-2 {
+  opacity: 0;
+}
+
```
Comment (sja-thedude) [inline style.css]: - [x] Kindly make sure the background image for the mobile menu is placed properly as per the Figma design.
- [x] Kindly make sure that the menu options are correctly placed with correct spacing as per the Figma design.
##  Make sure to follow the video attached to see where you have to make changes and make sure your design resembles Figma's design.

https://user-images.githubusercontent.com/70269507/169270596-07c7ab8a-4db1-47d9-bb18-f41b296867a4.mov

**Review by sja-thedude — CHANGES_REQUESTED** [review]
## Required Changes ♻️

Hi @Roseokpe, This is @sja-thedude your code reviewer.

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

768070f3-bc9a-4aba-b8bc-25793e00ea1e_text

## Highlights
✔️ Good job in implementing the mobile menu correctly
✔️ Correct git-flow

## Required 
- [ ] Kindly make sure that linters are correctly set up for this project. Make sure to replace .github/workflows -> linters.yml with [this](https://github.com/microverseinc/linters-config/blob/master/html-css-js/.github/workflows/linters.yml)
- [ ] Kindly update your readme according to the project you are working on.
- [ ] Kindly make sure to add setup details along with your details in the readme.
- [ ] _Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@sja-thedude] in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by OLIPLICHE — APPROVED** [review]
Hi @Roseokpe , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**